blocks/init/assets/styles/admin/_login.scss

```diff
@@ -0,0 +1,11 @@
+/* Full path due to wp-login.php is not in the theme folder */
+
+.login h1 a { /* stylelint-disable-line selector-no-type */
+  background-image: url('/wp-content/themes/eightshift-boilerplate/public/logo.svg');
```

Comment:
So what if you don't use the lib with our boilerplate? Maybe add this in a comment on how to add the image for the login? Because it's too specific for a lib imo.

Reply:
👍

Reply:
👍